engine/src/test/java/io/zeebe/engine/util/EngineRule.java

```diff
@@ -139,6 +144,29 @@ public void increaseTime(Duration duration) {
     environmentRule.getClock().addTime(duration);
   }
 
+  public void reprocess() {
```

Comment:
Maybe, we could rename it to `deleteSnapshotAndReprocess()`. What do you think?

Reply:
Reprocessing includes without snapshot since in this tests always the snapshots are enforced.